src/audio/mux/mux_generic.c

```diff
@@ -385,11 +385,17 @@ const struct comp_func_map mux_func_map[] = {
 
 mux_func mux_get_processing_function(struct comp_dev *dev)
 {
-	struct comp_data *cd = comp_get_drvdata(dev);
+	struct comp_buffer *sinkb;
 	uint8_t i;
 
+	if (list_is_empty(&dev->bsink_list))
+		return NULL;
+
+		sinkb = list_first_item(&dev->bsink_list, struct comp_buffer,
+					source_list);
```

Comment:
Indentation: Should be same as of the if statement above or the for loop below. Looks odd and might trip Checkpatch as well (didn't look)

Reply:
done